### PR TITLE
Eng 972 improve docs about bundle versions

### PR DESCRIPTION
**Pull request by Kerruba**

### CODE REVIEW

**Comment by Kerruba** [comment]
@nshaw and @joewhite101 I'm happy to move this to the more appropriate section, but under "docs" atm the informations avaiable are quite generic compared with the details reported in this documentation. If we want to move this into the `docs` section, I think we should probably move more sections over. What do you think?

**Comment by nshaw** [comment]
I agree with @joewhite101. It's not a completely clean separation but we're trying to have docs be the conceptual/why-we-do-things area and tutorials contain the step-by-step instructions on how to do something. If you see other items under tutorials that make more sense under docs, feel free to propose moving those as well.

**Comment by joewhite101** [comment]
Yeah, I think we should create a top level menu entry for the ECR under docs and move this one and also these:

ecr-troubleshooting-guide.md
ecr-overview.md
ecr-bundle-filters.md
ecr-bundle-details.md
ecr-uninstall-flow.md

@Kerruba if you want me to do that late me know. Otherwise we can resolve the conflicts, merge this, and then i'll do those moves in a second PR.

**Comment by nshaw** [comment]
@joewhite101 , I like that approach. Those files have definitely needed some reorganizing.

**Comment by Kerruba** [comment]
@joewhite101 please let's do that update first, then I'll update this PR and move it under the relevant part of the documentation

**Comment by firegloves** [comment]
@Kerruba update to Installable

File: vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+For example, let's assume my bundle already has a version `1.0.0` and I want to publish the version `2.0.0` of my bundle, here the commands I need to use to create and publish the new tag:
+
+```
+    git tag -a "2.0.0" -m "My new version"
+    git push --tags
+```
+
+## How to prevent a particular bundle version from being installable?
+
+If, for some reasons, you don't want a particular version to be available for the installation (anymore), you can proceed like follows:
```
Comment (Kerruba) [inline vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md:57]:
```suggestion
If, for some reasons, you don't want a particular version to be available for the install, you can proceed like follows:
```

File: vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+1. remove the undesired version from the tag list in your bundle Kubernetes file (`integrity`, `shasum`, `tarball` and `version`)
+2. delete your bundle from your cluster using a command like this `kubectl delete -f your-bundle-file.yml [-n your-cluster-namespace]`
+3. upload again your bundle to Kubernetes using a command like this `kubectl create -f your-bundle-file.yml [-n your-cluster-namespace]`
+
+## My bundle contains a microservice generated with the Entando Component Generator, does the version of the microservice have the same as the bundle version?
```
Comment (Kerruba) [inline vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md:63]:
```suggestion
## My bundle contains a microservice generated with the Entando Component Generator, does the version of the microservice have to be the same as the bundle version?
```

File: vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+The version of the microservice - or to be more precise the docker image associated with the microservice - are not directly releated to the version of the bundle containing the microservice.
```
Comment (Kerruba) [inline vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md:65]:
```suggestion
The version of the microservice - or to be more precise the docker image associated with the microservice - isn't bound to the version of the bundle containing the microservice itself. 
```

File: vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md (hunk 1 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md:57]: Small changes on this one. Reword and grammar

```
If, for some reason, you don't want a particular version to be available for installation, you can proceed as follows:

```

**Pull request closed without merging**